app/controllers/school_search_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "govspeak"
```

Comment:
Do we need this in here?

Reply:
Nope, copy-pasted the controller without thinking too much.